src/components/NcAppSidebar/NcAppSidebarTabs.vue

```diff
@@ -50,7 +50,7 @@
 				class="app-sidebar-tabs__tab"
 				:class="{ active: tab.id === activeTab }"
 				role="tab"
-				type="radio"
+				type="button"
```

Comment:
This is not valid, NcCheckboxRadioSwitch only accepts checkbox radio and switch als property values.

Reply:
Thats new :D